net-nntp/slrn/files/slrn-1.0.3-configure.patch

```diff
@@ -0,0 +1,75 @@
+https://bugs.gentoo.org/874687
+from
+https://github.com/jedsoft/slrn/pull/1
+edited to current situation (line numbers changed so patch applies cleanly)
+
+From f550d0ab9e1cd3cfe844022435c3d98af3c18441 Mon Sep 17 00:00:00 2001
+From: Florian Weimer <fweimer@redhat.com>
+Date: Sat, 4 Feb 2023 15:04:36 +0100
+Subject: [PATCH] autoconf: Fix C99 compatibility issue in the VA_COPY checks
+
+The exit function is called without including <stdlib.h>, resulting
+in an implicit function declarations.  Future compilers will not
+support implicit function declarations by default.  This will lead
+to build failures.
+---
+ autoconf/configure.ac | 3 +++
+ configure             | 3 +++
+ 2 files changed, 6 insertions(+)
```

Comment:
Hmm, you should pick one to patch. If patching **configure.ac** then remember to inherit *autotools.eclass* and call `eautoreconf` in `src_prepare()`.

Reply:
Upstream is one of those fascinating projects that:
- checks configure scripts into git
- distributes source code that *cannot* be `autoreconf`ed
- feels it is "unclean" to have configure.ac next to configure, so shoves it off into a subdir, which doesn't autoreconf either, but does autoconf.


I don't know that autotools.eclass can handle *any* of this.